src/PowerShell/Constants.ts

```diff
@@ -10,4 +10,5 @@ export default class Constants {
     static readonly Success: string = "Success";
     static readonly Error: string = "Error";
     static readonly AzVersion: string = "AzVersion";
+    static readonly  AzureSupportedCloudName:string[] = ["azureusgovernment", "azurechinacloud", "azuregermancloud","azurecloud","azurestack"];
```

Comment:
All the constants in this file are used only for PowerShell login. Move "AzureSupportedCloudName" into main.ts since "AzureSupportedCloudName" is used only for input validation

Reply:
also replace string array with Set<String>. usage reference:[https://alligator.io/js/sets-introduction/]( https://alligator.io/js/sets-introduction/ ), [APIs ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)